Userland/Games/Snake/SnakeGame.cpp

```diff
@@ -17,232 +17,239 @@
 
 SnakeGame::SnakeGame()
 {
-    set_font(Gfx::FontDatabase::default_fixed_width_font().bold_variant());
-    m_fruit_bitmaps.append(*Gfx::Bitmap::try_load_from_file("/res/icons/snake/paprika.png"sv).release_value_but_fixme_should_propagate_errors());
-    m_fruit_bitmaps.append(*Gfx::Bitmap::try_load_from_file("/res/icons/snake/eggplant.png"sv).release_value_but_fixme_should_propagate_errors());
-    m_fruit_bitmaps.append(*Gfx::Bitmap::try_load_from_file("/res/icons/snake/cauliflower.png"sv).release_value_but_fixme_should_propagate_errors());
-    m_fruit_bitmaps.append(*Gfx::Bitmap::try_load_from_file("/res/icons/snake/tomato.png"sv).release_value_but_fixme_should_propagate_errors());
-    reset();
-
-    m_high_score = Config::read_i32("Snake"sv, "Snake"sv, "HighScore"sv, 0);
-    m_high_score_text = String::formatted("Best: {}", m_high_score);
+   set_font(Gfx::FontDatabase::default_fixed_width_font().bold_variant());
+   m_food_bitmaps.append(*Gfx::Bitmap::try_load_from_file("/res/emoji/U+1F41F.png"sv).release_value_but_fixme_should_propagate_errors());
```

Comment:
It would be nice if we could somehow use actual emojis to load the files here, I personally cannot read unicode codepoints.

Anyway this is definitely not requirement, I'll have a think about this.

Reply:
This is 100% out of my skill or the scope of this PR :-)